examples/plt_cfg.yaml

```diff
@@ -19,77 +19,81 @@ results_config:
 taggers_config: !include taggers.yaml
 taggers: [dummy1, dummy2, dummy3]
 
-roc_plots:
+plots:
   - args:
       signal: cjets
+      plot_type: roc
       plot_kwargs:
```

Comment:
Just comming back to this, I'd probably keep the structure nested to avoid repition and to see organisation better. You also don't need the first `args:` key

```yaml
plots:
   roc:
    - signal: cjets
      kwargs:
        x_range: [0.5, 1.0]

```